src/__stories__/circle-story.tsx

```diff
@@ -1,27 +1,49 @@
 import * as React from 'react';
-import {Circle, skinVars, Stack, IconShopRegular} from '..';
+import {Circle, skinVars, IconShopRegular} from '..';
 import avatarImg from './images/avatar.jpg';
 
 export default {
     title: 'Components/Primitives/Circle',
+    argTypes: {
+        size: {
+            control: {type: 'range', min: 40, max: 128, step: 4},
+        },
+        content: {
+            options: ['color', 'image', 'icon', 'none'],
+            control: {type: 'select'},
+        },
+    },
 };
 
-export const Default: StoryComponent = () => {
-    return (
-        <Stack space={16} dataAttributes={{testid: 'circle'}}>
-            <Circle size={40} backgroundColor={skinVars.colors.brand} />
-
-            <Circle size={40} backgroundColor={skinVars.colors.brandLow}>
-                <IconShopRegular color={skinVars.colors.brand} />
-            </Circle>
-
-            <Circle size={40} backgroundImage={avatarImg}></Circle>
-
-            <Circle size={40} backgroundColor={skinVars.colors.background} border />
+type Args = {
+    content: string;
+    size: number;
+    border: boolean;
+};
 
-            <Circle size={40} backgroundColor={skinVars.colors.background} border="red" />
-        </Stack>
+export const Default: StoryComponent<Args> = ({content, size, border}) => {
+    return (
+        <Circle
+            size={size}
+            dataAttributes={{testid: 'circle'}}
```

Comment:
not needed, you could use the `data-component-name`

Reply:
Discussed offline. We decided to keep the data-testid, because we use this id specifically for testing purposes and it makes sense to differentiate it from the component name.